docs/pages/workflow/expo-cli.md

```diff
@@ -5,30 +5,41 @@ maxHeadingDepth: 4
 
 import { Terminal } from '~/ui/components/Snippet';
 
-Expo CLI is a command line app that is the main interface between a developer and Expo tools. You'll use it for a variety of tasks, such as:
+Expo CLI is the primary command-line interface between a developer and Expo tools. It is used for a variety of tasks, such as:
 
-- Creating new projects
-- Developing your app: running the project server, viewing logs, opening your app in a simulator
-- [Publishing](publishing.md) your app JavaScript and other assets and managing releasing them as updates
-- [Building binaries](/distribution/building-standalone-apps.md) (`aab` and `ipa` files) to be [uploaded to the App Store and Play Store](/distribution/uploading-apps.md)
+- To run a project's [development server](/guides/how-expo-works/#expo-development-server), view logs, open the app in a simulator, etc., during the development phase of an app
+- [Publishing](publishing) an app's JavaScript and other assets and managing to release them as updates
+- [Building binaries](/distribution/building-standalone-apps) (`.aab`, `.apk`, and `.ipa` files) and [uploading them to app stores](/distribution/uploading-apps)
```

Comment:
Even though Expo CLI is technically capable of building binaries in the cloud, we should only reference EAS Build/Submit.

Reply:
Maybe, we should remove the last point about building binaries?

If not, the other way I can think of is to adjust the text like this:

```markdown
- [Building binaries](/distribution/building-standalone-apps) (`.aab`, `.apk`, and `.ipa` files) and [uploading them to app stores](/distribution/uploading-apps) using Classic Build Service. Although, it is recommended to use EAS Build to build binaries and submit them to app stores with support for custom native code and smaller app archives.
```